Makefile

```diff
@@ -9,3 +9,20 @@ test-integration:
 
 test-sanity:
 	ansible-test sanity --docker -v --color --python $(PYTHON_VERSION) $(?TEST_ARGS)
+
+generate-api-client:
+	npx @openapitools/openapi-generator-cli generate \
+	-g python \
+	-i ./api/v1alpha1/openapi.yml \
+	-o ./lib/flightctl_api_client \
+	--global-property=apiDocs=false,modelDocs=false,apiTests=false,modelTests=false \
+	--additional-properties=generateSourceCodeOnly=true
+
+generate-sanity-ignore-file:
```

Comment:
This is due to the reality of having generated code in the ansible collection in this manner, as the ability to skip sanity tests [doesn't work with wildcards](https://github.com/ansible/ansible/issues/78346#issuecomment-1939191550) so to skip linting errors for the generated client we have to skip for each file.